api/envoy/extensions/tracers/opentelemetry/samplers/v3/dynatrace_sampler.proto

```diff
@@ -44,10 +44,10 @@ message DynatraceSamplerConfig {
 
   // Default number of root spans per minute, used when the value can't be obtained from the Dynatrace API.
   //
-  // A default value of `1000` is used when:
+  // A default value of 1000 is used when:
   //
-  // - `root_spans_per_minute` is unset
-  // - `root_spans_per_minute` is set to 0
+  // - root_spans_per_minute is unset
```

Comment:
```suggestion
  // - ``root_spans_per_minute is unset``
```

Reply:
Ah thanks, I didn't about the double backticks. Had single ones but the build was failing. Fixed in [106992a](https://github.com/envoyproxy/envoy/pull/32598/commits/106992a550961a7d099822e6e95809819836c851)